### PR TITLE
add Japanese version

### PR DESCRIPTION
**Pull request by toshi0607**
Your work is wonderful!!! I was so impressed that I translated the doc into Japanese.

### CODE REVIEW

File: ja-jp/README.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+
+たとえば、 `--restart-policy = Always` をリソースは Deployment リソースとみなされ、 `--restart-policy = Never` を持つリソースは Pod リソースとみなされます。kubectl はコマンドを記録する（ロールアウトや監査用）など他のアクションを起動する必要があるかどうか、このコマンドが単なるドライランであるかどうか（ `--dry-run` フラグが指定される）も判断します。
+
+Deployment リソースを作成したいことが認識された後、提供されたパラメータからランタイムオブジェクトを生成するために `DeploymentV1Beta1` ジェネレーターを使います。「ランタイムオブジェクト」はリソースの総称です。
+
+### APIグループとバージョンネゴシエーション
+
+先に進む前に指摘する価値があるのは、Kubernetes は「APIグループ」に分類される _versioned_API を使用しているということです。APIグループは、似たリソースを分類して、簡単に推測できるようにすることを目的としています。それはまた、単一のモノリシックAPIに対するより良い代替手段を提供します。Deployment リソースのAPIグループは `apps` という名前で、その最新バージョンは `v1` です。Deployment リソースのマニフェストの上部に `apiVersion: apps/v1` と書く必要があるのはこのためです。
+
+とにかく... kubectl はランタイムオブジェクトを生成した後、¥適切なAPIグループとそれに対するバージョンを見つけ始め](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/run.go#L580-L597)、リソースに対する様々なRESTセマンティクスを知っている[バージョン管理されたクライアント](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/run.go#L598)を組み立てます。この探索ステージはバージョンネゴシエーションと呼ばれ、すべての利用可能なAPIグループを取得するためにリモートAPI上の `/apis` パスを kubectl がスキャンすることを含みます。kube-apiserver はこのパスでスキーマ文書（ OpenAPI フォーマット）を公開しているので、クライアントがディスカバリーを実行するのは簡単です。
```
Comment (d-kuro) [inline ja-jp/README.md]: Unnecessary `¥` ?

```suggestion
とにかく... kubectl はランタイムオブジェクトを生成した後、適切なAPIグループとそれに対するバージョンを見つけ始め](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/run.go#L580-L597)、リソースに対する様々なRESTセマンティクスを知っている[バージョン管理されたクライアント](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/run.go#L598)を組み立てます。この探索ステージはバージョンネゴシエーションと呼ばれ、すべての利用可能なAPIグループを取得するためにリモートAPI上の `/apis` パスを kubectl がスキャンすることを含みます。kube-apiserver はこのパスでスキーマ文書（ OpenAPI フォーマット）を公開しているので、クライアントがディスカバリーを実行するのは簡単です。
```

File: ja-jp/README.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+1. 監査と最終アドミッションが[実行され](https://github.com/kubernetes/apiserver/blob/7001bc4df8883d4a0ec84cd4b2117655a0009b6c/pkg/endpoints/handlers/create.go#L93-L104)ます。
+1. [ストレージプロバイダに委譲する](https://github.com/kubernetes/apiserver/blob/19667a1afc13cc13930c40a20f2c12bbdcaaa246/pkg/registry/generic/registry/store.go#L327)ことでリソースが [etcd に保存され](https://github.com/kubernetes/apiserver/blob/7001bc4df8883d4a0ec84cd4b2117655a0009b6c/pkg/endpoints/handlers/create.go#L111)ます。通常、etcd キーは `<名前空間>/<名前>`の形式になりますが、設定可能です。
+1. あらゆる作成エラーがキャッチされ、最後に、ストレージプロバイダーはオブジェクトが実際に作成されたことを確認するために `get` 呼び出しを実行します。追加のファイナライズが必要な場合は、post-create ハンドラーとデコレータを呼び出します。
+1. HTTP レスポンスが[作成され](https://github.com/kubernetes/apiserver/blob/7001bc4df8883d4a0ec84cd4b2117655a0009b6c/pkg/endpoints/handlers/create.go#L131-L142)て返送されます。
+
+たくさんのステップがあります！私たちが実際にどれだけの仕事をしているのかを理解しているので、ウサギの穴のパンの耳をたどるのはとても素晴らしいことです。要約すると、Deployment リソースは etcd にあります。しかし、ざわと壊すだけでは、まだ見ることはできません…
```
Comment (d-kuro) [inline ja-jp/README.md]:
```suggestion
たくさんのステップがあります！私たちが実際にどれだけの仕事をしているのかを理解しているので、ウサギの穴のパンの耳をたどるのはとても素晴らしいことです。要約すると、Deployment リソースは etcd にあります。しかし、わざと壊すだけでは、まだ見ることはできません…
```

File: ja-jp/README.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+- [basicauth ハンドラー](https://github.com/kubernetes/apiserver/blob/51bebaffa01be9dc28195140da276c2f39a10cd4/plugin/pkg/authenticator/request/basicauth/basicauth.go#L37)は、HTTP リクエストの基本認証資格情報が自身のローカル状態と一致することを同様に保証します
+
+すべてのオーセンティケーターが失敗すると、[リクエストは失敗](https://github.com/kubernetes/apiserver/blob/20bfbdf738a0643fe77ffd527b88034dcde1b8e3/pkg/authentication/request/union/union.go#L71)し、集約エラーが返されます。認証が成功すると、 `Authorization` ヘッダがリクエストから削除され、ユーザ情報がそのコンテキストに追加されます。これにより、後続のステップ（認可や認可コントローラーなど）で以前に確立されたユーザーの ID にアクセスできるようになります。
+
+### 認可
+
+さて、リクエストは送信されました。そして kube-apiserver に認証されました。一安心です。しかしまだ終わっていません。私たちは認証されましたが、あるアクションを実行するための権限はあるのでしょうか？結局、アイデンティティと許可は同じものではありません。処理を続けるためには、kube-apiserver に認可される必要があります。
+
+kube-apiserver が認証を処理する方法は認証と非常に似ています。フラグ入力に基づき、すべてのリクエストに対して実行される一連のオーソライザーを集めます。すべてのオーソライザーがリクエストを拒否した場合、リクエストは `Forbidden` レスポンスとなり、それ以上[先に進むことはありません](https://github.com/kubernetes/apiserver/blob/e30df5e70ef9127ea69d607207c894251025e55b/pkg/endpoints/filters/authorization.go#L60)。単一のオーソライザーが認可するとリクエストが続行されます。
```
Comment (h-michael) [inline ja-jp/README.md]:
```suggestion
kube-apiserver が認可を処理する方法は認証と非常に似ています。フラグ入力に基づき、すべてのリクエストに対して実行される一連のオーソライザーを集めます。すべてのオーソライザーがリクエストを拒否した場合、リクエストは `Forbidden` レスポンスとなり、それ以上[先に進むことはありません](https://github.com/kubernetes/apiserver/blob/e30df5e70ef9127ea69d607207c894251025e55b/pkg/endpoints/filters/authorization.go#L60)。単一のオーソライザーが認可するとリクエストが続行されます。
```

File: ja-jp/README.md [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+- [webhook](https://github.com/kubernetes/apiserver/blob/d299c880c4e33854f8c45bdd7ab599fb54cbe575/plugin/pkg/authorizer/webhook/webhook.go#L143) はクラスタ外の HTTP（S） サービスとやり取りします
+- [ABAC](https://github.com/kubernetes/kubernetes/blob/77b83e446b4e655a71c315ad3f3890dc2a220ccf/pkg/auth/authorizer/abac/abac.go#L223) は静的ファイルで定義されたポリシーを強制します
+- [RBAC](https://github.com/kubernetes/kubernetes/blob/8db5ca1fbb280035b126faf0cd7f0420cec5b2b6/plugin/pkg/auth/authorizer/rbac/rbac.go#L43) は管理者によって k8s リソースとして追加された RBAC ロールを強制します
+- [NODE](https://github.com/kubernetes/kubernetes/blob/dd9981d038012c120525c9e6df98b3beb3ef19e1/plugin/pkg/auth/authorizer/node/node_authorizer.go#L67) ノードクライアント、つまり kubelet は自分自身がホストしているリソースにのみアクセスできるようになります
+
+各 `Authorize` メソッドをチェックしてそれらがどのように機能するのかを確かめてください！
+
+### アドミッションコントロール
+
+さて、それでこの時点で認証され、 kube-apiserver によって認可されました。それでは何が残っているでしょうか？kube-apiserver から見れば、私たちが誰であるかを信じ、継続することを許しますが、Kubernetesとしては、システムの他の部分は何が起こるべきか、そして許されるべきでないかについて強い意見を持ちます。ここで[アドミッションコントローラー](https://kubernetes.io/docs/admin/admission-controllers/#what-are-they)が登場します。
```
Comment (h-michael) [inline ja-jp/README.md]:
```suggestion
さて、それでこの時点で認証され、 kube-apiserver によって認可されました。それでは何が残っているでしょうか？kube-apiserver から見れば、私たちが何者であるかということを信頼し、継続することを許可しますが、Kubernetesとしては、システムの他の部分は何が起こるべきか、そして許されるべきでないかについて強い意見を持ちます。ここで[アドミッションコントローラー](https://kubernetes.io/docs/admin/admission-controllers/#what-are-they)が登場します。
```
これもあまりしっくりこないですが。

File: ja-jp/README.md [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+
+認可はユーザーが許可を持っているかどうかに答えることに焦点を当てていますが、アドミッションコントローラーはリクエストがクラスタのより広い期待とルールに一致することを保証するためにリクエストをインターセプトします。これらは、オブジェクトが etcd に永続化される前の制御の最後の砦です。アクションが予期しない結果や悪影響を与えないように、残りのシステムチェックをカプセル化します。
```
Comment (h-michael) [inline ja-jp/README.md]:
```suggestion
認可はユーザーが許可を得ているかどうかを答えることに焦点を当てていますが、アドミッションコントローラーはリクエストがクラスタのより広い期待とルールに一致することを保証するためにリクエストをインターセプトします。これらは、オブジェクトが etcd に永続化される前の制御の最後の砦です。アクションが予期しない結果や悪影響を与えないように、残りのシステムチェックをカプセル化します。
```

File: ja-jp/README.md [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+
+アドミッションコントローラーの動作方法はオーセンティケーターとオーソライザーの動作方法と似ていますが、1つの違います。オーセンティケーターおよびオーソライザーチェーンとは異なり、単一のアドミッションコントローラーが失敗すると、チェーン全体が壊れ、リクエストは失敗します。
```
Comment (h-michael) [inline ja-jp/README.md]:
```suggestion
アドミッションコントローラーの動作方法はオーセンティケーターとオーソライザーの動作方法と似ていますが、1つ違いがあります。オーセンティケーターおよびオーソライザーチェーンとは異なり、単一のアドミッションコントローラーが失敗すると、チェーン全体が壊れ、リクエストは失敗します。
```

**Comment by jamiehannaford** [comment]
Thank you so much!!!

**Pull request merged**